src/pages/groupComparison/GroupComparisonMutationsTab.tsx

```diff
@@ -41,13 +41,9 @@ export default class GroupComparisonMutationsTab extends React.Component<
     }
 
     @computed get tabs() {
-        return this.props.store.genesWithMaxFrequency.map(g => (
-            <MSKTab
-                key={g.hugoGeneSymbol}
-                id={g.hugoGeneSymbol}
-                linkText={g.hugoGeneSymbol}
-            />
-        ));
+        return this.props.store.genesWithHighestMutationFrequency
+            .result!.slice(0, 10)
+            .map(g => <MSKTab key={g} id={g} linkText={g} />);
```

Comment:
lets put prefix on both the key and id props.  e.g.  `comp-tab-${g}`
React keys need to be unique for whole page so best to prefix them in general

Reply:
added